test/process.test.js

```diff
@@ -94,7 +98,7 @@ describe(`process`, function () {
     let output = p.process(`<template>Hi</template>`, { inline_source_map: true });
 
     expect(output).to.match(
-      /sourceMappingURL=data:application\/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIjxhbm9uPiJdLCJzb3VyY2VzQ29udGVudCI6WyI8dGVtcGxhdGU-SGk8L3RlbXBsYXRlPiJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiO0FBQUEsZUFBQSxTQUFVLENBQUEsRUFBRSxDQUFBLEVBQUE7SUFBQTtRQUFBLE9BQUEsS0FBQSxTQUFBLENBQUEsRUFBVztJQUFEO0FBQUEsR0FBQyJ9/
+      /sourceMappingURL=data:application\/json;base64,/
```

Comment:
is this right? 🤔

Reply:
I know nothing about source maps. 

and... _currently_ our gjs/gts sourcemaps are very weird in-browser

Reply:
It's pretty hard to check the exact source map now that the template imports gets a UUID. This at least confirms that we did get an inline source map of some sort.

Reply:
This looks like an empty source map though

Reply:
Ah, got it